From d028ca8b26890eb4e5fac5db96479feefa86effa Mon Sep 17 00:00:00 2001
From: trainman419 <legotown@aol.com>
Date: Thu, 17 Apr 2014 15:55:09 -0700
Subject: [PATCH] I'm a terrible person?

---
 CMakeLists.txt |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 0cd7e9a..a527b0d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -15,6 +15,8 @@ ExternalProject_Add(gperftools
 catkin_package(
   )
 
+install()
+
 ExternalProject_Get_Property(gperftools BINARY_DIR)
 add_custom_target(gperftools-install gperftools
   COMMAND make -C ${BINARY_DIR} install)
-- 
1.7.9.5

